src/openrct2/network/network.cpp

```diff
@@ -784,6 +785,9 @@ void Network::LoadGroups()
 		}
 		json_decref(json);
 	}
+
+	// Host group should always contain all permissions.
+	group_list.at(0)->ActionsAllowed.fill(0xFF);
```

Comment:
@Broxzier would you be able to fix this? it'd be a bit of a hassle for me to pull your pr and then submit a pr again. I was just trying to help, not completely take over. (no offense, i'm just kinda busy with other stuff myself)